go-controller/pkg/ovn/ovn.go

```diff
@@ -36,6 +37,14 @@ type ServiceVIPKey struct {
 	protocol kapi.Protocol
 }
 
+// LoadBalancerConf contains the OVN based config for a LB
+type LoadBalancerConf struct {
+	// List of endpoints as configured in OVN, ip:port
+	Endpoints []string
+	// ACL configured for Rejecting access to the LB
+	RejectACL string
+}
+
```

Comment:
can we not export these fields and the structure itself? looks like this whole structure is used within the `ovn` package?

Reply:
will change